JPS_Ontology/ontology/ontobuiltenv/OntoBuiltEnv.csv

```diff
@@ -121,7 +121,9 @@ Pharmacy,Class,Medical,IS-A,,,,A healthcare facility where medicinal drugs are p
 BuildingFloors,Class,,,Property,,,A concept to describe amount of floors in a building.,https://www.theworldavatar.com/kg/ontobuiltenv/,Building floors
 hasBuildingFloors,Object Property,,,Property,BuildingFloors,exactly 1,A relationship between a property and its amount of floors.,https://www.theworldavatar.com/kg/ontobuiltenv/,has building floors
 hasFloorsValue,Data Property,,,BuildingFloors,integer,exactly 1,A relationship between the floors and its value.,https://www.theworldavatar.com/kg/ontobuiltenv/,has floors value
```

Comment:
What is the "value" of a (set of) floors? This is completely unclear, and the comment doesn't explain anything. Please clarify.

Reply:
it's a data property to describe the numerical value of NumberOfFloors, as NumberOfFloors is a class.